source/about/cloud-subscriptions.rst

```diff
@@ -4,27 +4,83 @@ Cloud subscriptions
+
+Billing period
+--------------
+
+After you begin your Mattermost Cloud paid subscription, your first charge is applied at the end of the calendar month. Subsequent billing periods begin at 12 AM UTC on the first day of each calendar month and end at 11:59 PM UTC on the final day of the same calendar month. Per-user pricing for each billing period will be based on the number of registered users.
+
+Invoices appear in the Customer Portal immediately after the billing period ends.
+
+It may take up to a day for our billing systems to calculate, charge your credit card, and issue your invoice. Once charges are completed, the invoice will show as *Paid*. The invoice is a downloadable PDF file.
+
+If there's a payment failure, your invoice will show *Payment Failed*. Please review the accuracy of your credit card information in **Payment Information**. Unresolved failed payments may result in a delinquency and an interruption to your subscription.
+
+Adding and removing users in the middle of a billing period
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you add new users in the middle of a billing cycle, you'll only be charged for the days that the user is registered.
+
+Here's an example:
+
+Your organization has a Mattermost Cloud workspace, and you're on a monthly subscription at the rate of $10 USD per user per month. You add a new user on the 11th day of a 30-day billing period, and that user remains registered for the remainder of the billing period.
+
+The prorated subscription cost for this user is determined by multiplying the monthly amount ($10 USD) by the number of days they are actively registered in the billing period (20) and dividing by the number of days in the month (30).
+
+The prorated subscription cost for the user added on day 11 is $6.67 USD.
+
+Similarly, if you make a registered user inactive in the middle of a billing cycle, we will calculate the prorated amount using the formula above to determine the prorated charge.
 
-- Subscription invoices
-- Billing history
-- Organization information
-- Credit Card payment methods
+Failed or late payments
+~~~~~~~~~~~~~~~~~~~~~~~
 
-Please see the `Cloud Billing <https://docs.mattermost.com/manage/cloud-billing.html>`__ documentation for more detailed information on the Cloud billing process.
+You'll be notified immediately of failed payments both in-product, with a banner, and via email. The notifications provide directions on how to update payment information. Once payment information is updated, you'll be charged right away for the amount owing from the previous billing period. If payment information is not updated and the account remains in arrears, here's what to expect:
+
+.. csv-table::
+   :header: "Time in arrears", "Action taken"
+
+   "7 days", "An email is sent to the workspace Admin with directions to update payment information."
```

Comment:
Actually as of right now, there isn't a system to handle delinquent accounts. No emails are sent as well. This is something the Self-Serve team is going to be working on really soon though.

Reply:
This is really old content from way before you started (just for context!). Should we delete this or leave it?

Reply:
Yes let's delete it for now. For what it's worth, working on the automated delinquent system is something the self-serve team will be working on soon.

[MM-43398](https://mattermost.atlassian.net/browse/MM-43398?atlOrigin=eyJpIjoiNzYyNTJhMWEyYzgzNDUzY2IxMzllNDg3YzY2ZGE5Y2EiLCJwIjoiaiJ9)